scenarioo-client/app/components/detailarea/detail-accordion/detail-accordion.component.ts

```diff
@@ -57,15 +60,21 @@ export class DetailAccordionComponent {
     @Input()
     labelConfigurations: LabelConfigurationMap;
 
+    isAccordionCollapsed: boolean = false;
+
     constructor(private selectedBranchAndBuildService: SelectedBranchAndBuildService,
                 private locationService: LocationService,
-                private sketchIdsResource: SketchIdsResource) {
+                private sketchIdsResource: SketchIdsResource,
+                private localStorageService: LocalStorageService) {
     }
 
     ngOnInit(): void {
-        if (this.isFirstOpen === false) {
-            this.isAccordionCollapsed = true;
-        }
+        this.isAccordionCollapsed = this.localStorageService.getBoolean(MAIN_METADATA_SECTION_EXPANDED + this.key, this.isFirstOpen);
+    }
+
+    toggleAccordionCollapsedValue() {
+        this.isAccordionCollapsed = this.isAccordionCollapsed !== true;
```

Comment:
this.isAccordionCollapsed = !this.isAccordionCollapsed;
would be easier to understand